server/resp/src/main/java/org/infinispan/server/resp/RespModuleLifecycle.java

```diff
@@ -1,63 +1,20 @@
-import org.infinispan.server.resp.json.JsonSetFunction;
-import org.infinispan.server.resp.json.JsonStringAppendFunction;
-import org.infinispan.server.resp.json.JsonToggleFunction;
-import org.infinispan.server.resp.json.JsonTypeFunction;
 
 @InfinispanModule(name = "resp", requiredModules = "core")
 public class RespModuleLifecycle implements ModuleLifecycle {
 
    @Override
    public void cacheManagerStarting(GlobalComponentRegistry gcr, GlobalConfiguration globalConfiguration) {
-      final Map<Integer, AdvancedExternalizer<?>> externalizerMap = globalConfiguration.serialization()
-            .advancedExternalizers();
-
-      externalizerMap.put(EventListenerKeysFilter.EXTERNALIZER.getId(), EventListenerKeysFilter.EXTERNALIZER);
-      externalizerMap.put(WATCH.EXTERNALIZER.getId(), WATCH.EXTERNALIZER);
-      externalizerMap.put(EventListenerConverter.EXTERNALIZER.getId(), EventListenerConverter.EXTERNALIZER);
-      externalizerMap.put(ComposedFilterConverter.EXTERNALIZER.getId(), ComposedFilterConverter.EXTERNALIZER);
-      externalizerMap.put(JsonGetFunction.EXTERNALIZER.getId(), JsonGetFunction.EXTERNALIZER);
-      externalizerMap.put(JsonSetFunction.EXTERNALIZER.getId(), JsonSetFunction.EXTERNALIZER);
-      externalizerMap.put(JsonLenFunction.EXTERNALIZER.getId(), JsonLenFunction.EXTERNALIZER);
-      externalizerMap.put(JsonTypeFunction.EXTERNALIZER.getId(), JsonTypeFunction.EXTERNALIZER);
-      externalizerMap.put(JsonDelFunction.EXTERNALIZER.getId(), JsonDelFunction.EXTERNALIZER);
-      externalizerMap.put(JsonArrayAppendFunction.EXTERNALIZER.getId(), JsonArrayAppendFunction.EXTERNALIZER);
-      externalizerMap.put(JsonStringAppendFunction.EXTERNALIZER.getId(), JsonStringAppendFunction.EXTERNALIZER);
-      externalizerMap.put(JsonToggleFunction.EXTERNALIZER.getId(), JsonToggleFunction.EXTERNALIZER);
-      externalizerMap.put(JsonObjkeysFunction.EXTERNALIZER.getId(), JsonObjkeysFunction.EXTERNALIZER);
-      externalizerMap.put(JsonNumIncrByFunction.EXTERNALIZER.getId(), JsonNumIncrByFunction.EXTERNALIZER);
-      externalizerMap.put(JsonArrindexFunction.EXTERNALIZER.getId(), JsonArrindexFunction.EXTERNALIZER);
-      externalizerMap.put(JsonBucket.EXTERNALIZER.getId(), JsonBucket.EXTERNALIZER);
-
-      // Externalizer that could be loaded by other modules.
-      externalizerMap.put(ITERATION_FILTER, new IterationFilter.IterationFilterExternalizer());
-
       SerializationContextRegistry ctxRegistry = gcr.getComponent(SerializationContextRegistry.class);
-      ctxRegistry.addContextInitializer(SerializationContextRegistry.MarshallerType.GLOBAL, new PersistenceContextInitializerImpl());
-      ctxRegistry.addContextInitializer(SerializationContextRegistry.MarshallerType.PERSISTENCE, new PersistenceContextInitializerImpl());
+      ctxRegistry.addContextInitializer(SerializationContextRegistry.MarshallerType.GLOBAL, new org.infinispan.server.resp.PersistenceContextInitializerImpl());
```

Comment:
is it intentional adding `PersistenceContextInitializerImpl` to `MarshallerType.GLOBAL`?